cv32e40s/docs/VerifPlans/Simulation/xsecure/xsecure_vplan.csv

```diff
@@ -69,7 +69,7 @@ a_xsecure_hint_updates_lfsr2.",,,,,,,,,,,,,,,,,
 ,,No suppresion,ECC checking shall be performed for every actual register file read and the check result shall never be suppressed (also not if a register file bypass value is used instead of the value from the register file).,"Assert that any mismatch on a read results in an alert, if the error is of 1 or 2 bits.",Assertion check,Constrained-Random,Assertion Coverage,"A: a_glitch_xsecure_rf_ecc_rs1_bit_fault, a_glitch_xsecure_rf_ecc_rs2_bit_fault",,,,,,,,,,,,,,,,,
 ,,Reset value,Upon reset the register file general purpose registers shall be set to 32’b0 and the syndromes shall be set to a matching ECC value (such that no error is triggered by default).,Assert reset value is legal,Assertion check,"ENV capability, not specific test",Assertion Coverage,"A: a_xsecure_rf_ecc_gpr_reset_value, a_xsecure_rf_ecc_gpr_reset_value_rvfi_rs1, a_xsecure_rf_ecc_gpr_reset_value_rvfi_rs2",,,,,,,,,,,,,,,,,
 ,,Syndrome non-readable,No explicit direct access mechanism shall be provided for the register file syndromes,"Code inspection: check the register file, and make sure there are no signals from the register syndrome out of the module",N/A,N/A,N/A,N/A,,,,,,,,,,,,,,,,,
-,,All 1's or all 0's is an error,The ECC codes shall be such that a value of all 1s or all 0s for a 32-bit word and its associated syndrome represent ECC errors.,"Verify that the syndrom of an register with value all 1s, is not all 1s, and the same for the case with all 0s. If the syndrom and register word is all 1s or all 0s, verify that the major alert is set.",Assertion check,Constrained-Random and Directed Self-Checking,Assertion Coverage,"A: a_xsecure_rf_ecc_gpr_0_syndrome_not_0, a_xsecure_rf_ecc_syndrome_0_gpr_not_0, a_xsecure_rf_ecc_gpr_1_syndrome_not_1, a_xsecure_rf_ecc_syndrome_1_gpr_not_1, a_glitch_xsecure_rf_ecc_gpr_0_syndrom_0, a_glitch_xsecure_rf_ecc_gpr_1_syndrom_1",,,,,,,,,,,,,,,,,
+,,All 1's or all 0's is an error,The ECC codes shall be such that a value of all 1s or all 0s for a 32-bit word and its associated syndrome represent ECC errors.,"Verify that the syndrom of an register with value all 1s, is not all 1s, and the same for the case with all 0s. If the syndrom and register word is all 1s or all 0s, verify that the major alert is set.",Assertion check,Constrained-Random and Directed Self-Checking,Assertion Coverage,A: a_xsecure_rf_ecc_gpr_not_all_0s_or_1s,,,,,,,,,,,,,,,,,
```

Comment:
Why was the change in the assertion only that a name changed but the change in the vplan is much bigger?

Is the previous link to cover no longer relevant?

Reply:
I saw that the vplan was outdated. So when the assertion was changed for the first time I forgot to update the vplan.

Reply:
Ok thanks. (I wish linkage checking was done programmatically and automatically, but alas.)